util/testutil/integration/standalone.go

```diff
@@ -0,0 +1,106 @@
+package integration
+
+import (
+	"bufio"
+	"bytes"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"testing"
+	"time"
+)
+
+func init() {
+	register(&standalone{})
+}
+
+type standalone struct {
```

Comment:
I think we should rename `standalone` to `oci` or `runc`, but can be another PR.

Reply:
I agree. I think `oci` is correct here. But let's do it in all packages together in a separate PR.